components/react/src/Components/Project/Project.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import Tag from '../Tag';
 
-const Project = ({ name, description, slack_channel, tasks, taxonomies, additional_info }) => {
+const Project = ({ name, search, description, slack_channel, tasks, taxonomies, additional_info }) => {
+  if (name.toLowerCase().match(search.toLowerCase()) !== null) {
```

Comment:
```suggestion
  if (name.toLowerCase().match(search.toLowerCase()) === null) return <></>;
  
```
Otherwise I would fix the indentation below.